package.json

```diff
@@ -17,6 +17,7 @@
     "eth-ens-namehash": "^2.0.8",
     "history": "^4.7.2",
     "js-sha3": "^0.8.0",
+    "lodash": "^4.17.14",
```

Comment:
Why is `lodash` added?

Reply:
hahaha, I dunno, that was from your commit! I'll remove it!

Reply:
Jaja right... fixing it